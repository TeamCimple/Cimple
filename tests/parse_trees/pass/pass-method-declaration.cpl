struct MyStruct {
	int a;
	int b;
};

int (MyStruct *m) getA() {
	return m.getA();
}
