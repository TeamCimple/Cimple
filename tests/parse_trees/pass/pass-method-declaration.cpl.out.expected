Struct((int Var(a),), (int Var(b),), , MyStruct, Constructor(Body:
        CompoundStatement(Declarations:  StatementList: )),, )FuncDecl(Name: Var(getA) ReturnType: RECEIVER(MyStruct,m) int Parameters:  Body: CompoundStatement(Declarations:  StatementList: RETURN(Call(Receiver(Identifier(m))FunctionName: getA Params: )))) 
